sdks/python/apache_beam/coders/avro_coder.py

```diff
@@ -1,99 +0,0 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-"""Coder for AvroRecord serialization/deserialization."""
-
-from __future__ import absolute_import
-
-import json
-from io import BytesIO
-
-from fastavro import parse_schema
-from fastavro import schemaless_reader
-from fastavro import schemaless_writer
-
-from apache_beam.coders.coder_impl import SimpleCoderImpl
-from apache_beam.coders.coders import Coder
-from apache_beam.coders.coders import FastCoder
-
-AVRO_CODER_URN = "beam:coder:avro:v1"
-
-__all__ = ['AvroCoder', 'AvroRecord']
```

Comment:
FYI: I think it's OK to remove this since it was just added two months ago but for future, please make sure to add the experiment annotations till APIs are stabilized.